DigitalLearningSolutions.Data.Migrations/Scripts/DLSV2-581-GetActiveAvailableCustomisationsForCentreFiltered_V6.sql

```diff
@@ -0,0 +1,99 @@
+/****** Object:  StoredProcedure [dbo].[GetActiveAvailableCustomisationsForCentreFiltered_V6]    Script Date: 29/09/2022 19:11:04 ******/
```

Comment:
@manishagarwalnhs, we need a Migration to add this SP to the DB. See `\DigitalLearningSolutions.Data.Migrations\202104220853_FixCustomisationUpdateSPs.cs` for an example of a migration that updates a SP.

Reply:
and this one that adds a new sp: `DigitalLearningSolutions.Data.Migrations\202101041622_ReorderFrameworkCompetenciesAndGroupsSPs.cs`

Reply:
This is done and checked in